provisioners/step.go

```diff
@@ -212,14 +202,16 @@ func decodeCSR(data []byte) (*x509.CertificateRequest, error) {
 	return csr, nil
 }
 
-// encodeX509 will encode a *x509.Certificate into PEM format.
-func encodeX509(cert *x509.Certificate) ([]byte, error) {
-	caPem := bytes.NewBuffer([]byte{})
-	err := pem.Encode(caPem, &pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw})
-	if err != nil {
-		return nil, err
+// encodeX509 will encode a certificate into PEM format.
+func encodeX509(certs ...capi.Certificate) ([]byte, error) {
+	certPem := bytes.NewBuffer([]byte{})
+	for _, cert := range certs {
+		err := pem.Encode(certPem, &pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw})
```

Comment:
When you encode like this, does it just append the new bytes on to the end of the existing array? I would have though it overwrites it.

Reply:
certPem is a `*bytes.Buffer`, here we're using it as an `io.Writer`, so always adding the new cert at the end.